app/src/main/java/com/architectcoders/arquitectomarvel/ui/detail/HeroDetailActivity.kt

```diff
@@ -89,12 +104,16 @@ class HeroDetailActivity : AppCompatActivity() {
                     android.R.drawable.star_off,
                     this.isCharacterFavorite
                 )
-                listener(character, adapter.currentList, this.isCharacterFavorite)
+                listener(
+                    character.fromResultUItoCharacterResult(),
+                    adapter.currentList,
+                    this.isCharacterFavorite
+                )
             }
         }
     }
 
-    private fun updateComics(comicList: List<ComicResult>) {
+    private fun updateComics(comicList: List<com.architectcoders.arquitectomarvel.data.remote.models_moshi.comics.Result>) {
```

Comment:
@joseantoniocampillo could you add an import alias?

Reply:
done